example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -43,11 +43,11 @@ describe('The S3 Ingest Granules workflow', () => {
       lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'SyncGranule');
     });
 
-    it('has expected payload', () => {
+    it('outputs 1 granule and 2 files in the payload', () => {
       expect(lambdaOutput.payload).toEqual(expectedSyncGranulePayload);
```

Comment:
No need to change (since you didn't write it in this one), but it'd probably be better to just have the actual assertions you're claiming to test here, i.e.

    expect(lambdaOutput.payload.granule.length).toEqual(1);
    expect(lambdaOutput.payload.files.length).toEqual(2);

or whatever those fields are.